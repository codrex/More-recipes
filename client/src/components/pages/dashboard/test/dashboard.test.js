import React from 'react';
import Dashboard from '../dashboard';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import configureStore from '../../../../store/configStore';


describe('Dashboard component ', () => {
  test('render as expected ', () => {
    const component = renderer.create(
      <Provider store={configureStore()}>
        <BrowserRouter>
          <Dashboard user={{}} />
        </BrowserRouter>
      </Provider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
