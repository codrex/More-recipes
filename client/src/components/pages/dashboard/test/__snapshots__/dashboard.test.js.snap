// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component  render as expected  1`] = `
<div
  className="container-fluid dashboard"
  id="dashboard"
>
  <div
    className="overlay false"
  >
    <div
      className="icon-wrapper mobile right animate"
      onClick={[Function]}
      onMouseEnter={undefined}
    >
      <i
        className="icon fa fa-times "
        data-target={undefined}
        data-toggle={undefined}
      >
        <span />
      </i>
    </div>
  </div>
  <div
    className="row dashboard"
  >
    <div
      className="col-xs-12 col-sm-12 col-md-3 col-lg-2 sidebar-wrapper d-flex false"
    >
      <div
        className="sidebar col-12 "
      >
        <div
          className="sidebar-header"
        >
          <div
            className="avatar avatar-sm"
          >
            
          </div>
          <h4
            className="lead sidebar-text"
          >
            Dashboard
          </h4>
          <a
            href="/user"
            onClick={[Function]}
          >
            <p
              className="small text-white text-capitalize"
            >
              my profile
            </p>
          </a>
        </div>
        <div
          className="sidebar-body"
        >
          <h4
            className="lead sidebar-text sidebar-list-item active"
            onClick={[Function]}
          >
            All recipes
          </h4>
          <h4
            className="lead sidebar-text sidebar-list-item"
            onClick={[Function]}
          >
            Top recipes
          </h4>
          <h4
            className="lead sidebar-text sidebar-list-item"
            onClick={[Function]}
          >
            Favorite recipes
          </h4>
        </div>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-12 col-md-9 col-lg-10
            recipes-wrapper d-flex no-padding align-items-center"
    >
      <div
        className="col-12 top-bar justify-content-end"
      >
        <div
          className="icon-wrapper mobile left"
          onClick={[Function]}
          onMouseEnter={undefined}
        >
          <i
            className="icon fa fa-bars"
            data-target={undefined}
            data-toggle={undefined}
          >
            <span />
          </i>
        </div>
        <form
          className="searchbox"
          onSubmit={[Function]}
        >
          <input
            className="form-control text-input'"
            onChange={[Function]}
            placeholder="Search"
            type="search"
            value=""
          />
          <div
            className="icon-wrapper"
            onClick={undefined}
            onMouseEnter={undefined}
          >
            <i
              className="icon fa fa-search"
              data-target={undefined}
              data-toggle={undefined}
            >
              <span />
            </i>
          </div>
        </form>
      </div>
      <div
        className="row grid-list-wrapper d-flex col-md-12 col-lg-12"
      >
        <div
          className="col-12 grid"
        >
          <div
            className="display-3 text-uppercase msg-display no-margin"
          >
            <h1>
              Sorry..
              <br />
              No recipe was found.
            </h1>
          </div>
        </div>
        <div
          className="row pagination-wrapper d-flex"
        />
      </div>
    </div>
  </div>
</div>
`;
